src/main/java/org/alfresco/rest/api/nodes/NodesEntityResource.java

```diff
@@ -1,193 +1,203 @@
-     * 
-     * @param fileNodeId
-     * @param contentInfo Basic information about the content stream
-     * @param stream An inputstream
-     * @param parameters
-     * @return
-     */
-    @Override
-    @WebApiDescription(title = "Upload content", description = "Upload content")
-    @BinaryProperties({"content"})
-    public Node updateProperty(String fileNodeId, BasicContentInfo contentInfo, InputStream stream, Parameters parameters)
-    {
-        return nodes.updateContent(fileNodeId, contentInfo, stream, parameters);
-    }
-
-    /**
-     * Update info on the node 'nodeId' - folder or document
-     *
-     * Can update name (which is a "rename" and hence must be unique within the current parent folder)
-     * or update other properties.
-     *
-     * @param nodeId  String nodeId of node (folder or document)
-     * @param nodeInfo node entity with info to update (eg. name, properties ...)
-     * @param parameters
-     * @return
-     */
-    @Override
-    @WebApiDescription(title="Updates a node (file or folder) with id 'nodeId'")
-    public Node update(String nodeId, Node nodeInfo, Parameters parameters)
-    {
-        return nodes.updateNode(nodeId, nodeInfo, parameters);
-    }
-    
-    /**
-     * Delete the given node. Note: will cascade delete for a folder.
-     * 
-     * @param nodeId String id of node (folder or document)
-     */
-    @Override
-    @WebApiDescription(title = "Delete Node", description="Delete the file or folder with id 'nodeId'. Folder will cascade delete")
-    public void delete(String nodeId, Parameters parameters)
-    {
-        nodes.deleteNode(nodeId, parameters);
-    }
-
-    @Operation("copy")
-    @WebApiDescription(title = "Copy Node", description="Copy one or more nodes (files or folders) to a new target folder, with option to rename.")
-    public Node copyById(String nodeId, NodeTarget target, Parameters parameters, WithResponse withResponse)
-    {
-       return nodes.moveOrCopyNode(nodeId, target.getTargetParentId(), target.getName(), parameters, true);
-    }
-
-    @Operation("move")
-    @WebApiDescription(title = "Move Node",
-            description="Moves one or more nodes (files or folders) to a new target folder, with option to rename.",
-            successStatus = HttpServletResponse.SC_OK)
-    public Node moveById(String nodeId, NodeTarget target, Parameters parameters, WithResponse withResponse)
-    {
-        return nodes.moveOrCopyNode(nodeId, target.getTargetParentId(), target.getName(), parameters, false);
-    }
-
-    @Operation("lock")
-    @WebApiDescription(title = "Lock Node",
-            description="Places a lock on a node.",
-            successStatus = HttpServletResponse.SC_OK)
-    public Node lock(String nodeId, LockInfo lockInfo, Parameters parameters, WithResponse withResponse)
-    {
-        return nodes.lock(nodeId, lockInfo, parameters);
-    }
+/*
```

Comment:
Most of the file is showing as changed when only a few lines have been added!?

Reply:
Some old files have a different line ending and is automatically changed to the default of the repository